resources/assets/components/Users/UserInformation.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
+import gql from 'graphql-tag';
 import { isEmpty } from 'lodash';
-import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/react-hooks';
 
-/* eslint-disable react/prop-types */
+export const UserInformationFragment = gql`
+  fragment UserInformation on User {
+    id
+    displayName
+    # age
```

Comment:
I forgot to add this to our GraphQL schema, hence why it's commented out for now! 🙈

Reply:
And uncommented in 38a0291.